### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Almightywesley**
Changes:
 * Added this amazing feature
 * Fixed bug where thing happened instead of thing
 * Improved this feature

How to test: 
 1. Do this
 2. Then do this

Screenshots:
If applicable, add a screenshot or video here to show the new feature

### CODE REVIEW

**Comment by MrTheGood** [comment]
I don't see any useful changes.

**Comment by Almightywesley** [comment]
accepteer man

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -19,13 +18,10 @@
...
-      <option name="XML_ALIGN_ATTRIBUTES" value="false" />
-      <option name="XML_SPACE_INSIDE_EMPTY_TAG" value="true" />
       <option name="XML_WHITE_SPACE_AROUND_CDATA" value="2" />
       <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
     </XML>
     <codeStyleSettings language="XML">
-      <option name="FORCE_REARRANGE_MODE" value="1" />
       <indentOptions>
         <option name="CONTINUATION_INDENT_SIZE" value="4" />
       </indentOptions>
```
Comment (MrTheGood) [inline .idea/codeStyles/Project.xml:27]: If you revert this file

Comment (Almightywesley) [reply .idea/codeStyles/Project.xml:27]: oke

**Pull request closed without merging**